src/components/hue/hue.component.tsx

```diff
@@ -21,15 +22,16 @@ export const Hue = memo(({ color, onChange }: IHueProps) => {
   }, [color.hsv.h, width]);
 
   const updateColor = useCallback(
-    (x: number) => {
+    (x: number, _y: number, final?: boolean) => {
```

Comment:
Given the changes to the `onCoordinateChange` event, `final` is now the first argument and no longer optional, and we don't need `_y` anymore.
We now need to apply these changes to the remaining components: `Saturation`, `Alpha`, and `Fields`.